ide.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should this be git ignored? If not, could you add usage docs to the contributing section?

Reply:
This was intentional.

It does not do anything outside of phpstorm, but it is a staring idea and currently provides autocomplete for TwillNavigation:

<img width="438" alt="Screenshot 2022-09-26 at 12 34 31" src="https://user-images.githubusercontent.com/866743/192255776-e308aad7-8144-4e52-996f-49529dd6f02f.png">

Reply:
Very nice! Thanks for clarifying.